src/main/java/io/quarkus/gizmo/MethodCreatorImpl.java

```diff
@@ -119,6 +139,11 @@ public void write(ClassVisitor file) {
                 av.visitEnd();
             }
         }
+        if (parameterNames != null) {
+            for (String parameterName : parameterNames) {
+                visitor.visitParameter(parameterName, 0);
```

Comment:
@Ladicek The javadoc of `MethodVisitor.visitParameter()` states that `only {@code ACC_FINAL}, {@code ACC_SYNTHETIC}    or/and {@code ACC_MANDATED} are allowed`. What does the zero represent?

Reply:
Those are bit flags, so `0` means no flag set. As in, not final, not synthetic and not mandated.